hipe4ml/tree_handler.py

```diff
@@ -145,6 +172,28 @@ def apply_preselections(self, preselections, inplace=True, **kwds):
         self._preselections = preselections
         return self._full_data_frame.query(preselections, inplace=inplace, **kwds)
 
+    def apply_model_handler(self, model_handler, output_margin=True):
+        """
+        Apply the ML model to data: a new column named model_output is added to the DataFrame
+        If a list is given the application is performed on the slices.
+
+        Parameters
+        ------------------------------------------------
+        model_handler: list or hipe4ml model_handler
+            If a list of handlers(one for each bin) is provided, the ML
+            model is applied to the slice
+
+        output_margin: bool
+            Whether to output the raw untransformed margin value.
+        """
+        if isinstance(model_handler, list):
+            for sliced_df in self._sliced_df_list:
+                sliced_df["model_output"] = model_handler.predict(
```

Comment:
This works only in the case of the binary classification. To make it work also in case of multi-class classification, one should add more columns (one for each class) with a for loop.